packages/forge/blocks/zendeskChat/index.ts

```diff
@@ -0,0 +1,14 @@
+import { createBlock } from '@typebot.io/forge'
+import { ZendeskChatLogo } from './logo'
+import { auth } from './auth'
+import { openMessenger } from './actions/openMessenger'
+import { authenticate } from './actions/authenticate'
+
+export const zendeskChatBlock = createBlock({
+  id: 'zendesk-chat',
+  name: 'Zendesk Chat',
```

Comment:
Can we rename it to just "Zendesk" I might add more actions in the future that are not tied to Chat

Reply:
Also change id to zendesk. Make sure to search and replace in the entire project 🙏

Reply:
I originally wanted to do exactly that, however, there are two separate API families for Zendesk, the core Zendesk API, https://vorihealth.zendesk.com/admin/apps-integrations/apis/zendesk-api/settings, which covers their Tickets, and other CRM functions and and their Conversations API -  https://vorihealth.zendesk.com/admin/apps-integrations/apis/conversations-api, which is the Sunshine conversations product they acquired. They are really two separate apps, and don't share API keys, etc. So, now sure if it would be confusing to do this and manage credentials as one single integration. What do you think?

Reply:
I updated it to Zendesk Messaging to be more aligned to the product. If there is a way to manage multiple credentials without confusing the user, happy to move to just Zendesk. Thoughts?

Reply:
I honestly think it makes more sense to name it Zendesk and update the credentials form accordingly, explaining that signing key is only useful for authenticating users before opening up the Zendesk Chat widget and that it is optional.

In the future we will add the fields for the other Zendesk APIs all of which will be optional as well.

Does that make sense?

Reply:
Makes sense to me. What I'm going to do is close this PR so I can generate the block properly as Zendesk. I'll add a new PR for that. Thanks!